docs/getting-started/overview.rst

```diff
@@ -1,6 +1,92 @@
-.. include:: /stub-topic.txt
+Overview of .NET implementations
+================================
+
+.NET Platform is, at its very core, a set of standards that can be implemented. There are 
+various implementations, some coming from Microsoft, some coming from other companies and 
+groups. This document will outline the current state of the .NET ecosystem and its various 
+implementations, so you can get a good "lay of the land". 
+
+.NET Framework
+--------------
+
+The .NET Framework is the premier implemetentation of the .NET Platform available for 
+Windows server and client developers. It is a very powerful, very mature framework, with 
+a huge class library (known as the **Base Class Library** or BCL for short) that supports 
+a wide variety of applications and solutions on Windows. 
+
+The additional stacks built on top the .NET Framework allow developers to build software 
+from CLI tools to rich web applications. 
```

Comment:
Need to define the acronym. The usual approach is "Command Line Interface (CLI)". That said, in our ecosystem, we call them console apps. That's the template name in VS.